servers/shareable-lists-api/src/models/ShareableListItem.ts

```diff
@@ -0,0 +1,48 @@
+import { SelectQueryBuilder } from 'kysely';
+import { DB } from '.kysely/client/types';
+import { ListItemResponse, ListItemSelect } from '../database/types';
+import { BaseContext, Connection } from '../shared/types';
+import { OffsetPagination } from '../database/OffsetPagination';
+import { ValidatedPagination } from '@pocket-tools/apollo-utils';
+
+export class ListItemModel {
+  constructor(private context: BaseContext) {}
+  byListId(
+    listId: number,
+  ): SelectQueryBuilder<DB, 'ListItem', ListItemResponse> {
+    const qb = this.context.conn
+      .selectFrom('ListItem')
+      .select(ListItemSelect)
+      .orderBy('sortOrder', 'asc')
+      .orderBy('createdAt', 'asc')
+      // Temporary tiebreaker sort for deterministic response
```

Comment:
whats the long term?

Reply:
either changing the sortOrder field so it doesn't allow collisions if sortOrder isn't explicitly set, or staying with this (or any other numeric ID)